src/lib/openapi/spec/environment-project-schema.ts

```diff
@@ -8,22 +8,37 @@ export const environmentProjectSchema = {
     properties: {
         name: {
             type: 'string',
+            example: 'development',
+            description: 'The name of the environment',
         },
         type: {
             type: 'string',
+            example: 'production',
+            description:
+                'The type of environment. Typically either development (non-production) or production',
         },
         enabled: {
             type: 'boolean',
+            example: true,
+            description:
+                '`true` if the environment is enabled for the project, otherwise `false`',
         },
         protected: {
             type: 'boolean',
+            example: false,
+            description:
+                '`true` if the environment is not removable, otherwise `false`',
         },
         sortOrder: {
             type: 'number',
+            example: 1,
+            description:
+                'Priority of the environment in a list of environments, the lower value, the higher up in the list the environment will appear',
         },
         projectApiTokenCount: {
             type: 'number',
             nullable: true,
+            description: 'How many API tokens are register',
```

Comment:
So this field can be a number OR null OR undefined (equivalent to `number | null | undefined` in TS)? When is it anything but a number? That should be mentioned in the description.

Also, I'm not sure I understand the description here. What does it mean? And what kind of API tokens does it count?

This suggestion needs to be reworked before it's ready.

```suggestion
            nullable: true,
            description: 'The number of client API tokens that have access to this project. `null` if .... `undefined` if ...'.
```